templates/patterns/inpage_navigation/pattern-inpage-navigation.html.twig

```diff
@@ -0,0 +1,16 @@
+{#
+/**
+ * @file
+ * Inpage navigation pattern.
+ */
+#}
+{{ attach_library('oe_bootstrap_theme/drupal.inpage_navigation') }}
+
+{% include '@oe-bcl/bcl-inpage-navigation/inpage-navigation.html.twig' with {
+  'title': title,
+  'full_layout': full_layout,
+  'links': links,
+  'content': content,
+  'id': scrollspy_id,
+  'attributes': attributes,
```

Comment:
we should add this only if it fixes something (and even so use create_attribute), but I don't think we need to be fixing anything here

Reply:
OK, removing it, but please read comment: https://github.com/openeuropa/oe_bootstrap_theme/pull/137#discussion_r765048075 because is not good to have different criteria on whether or not add attributes between team members.